README.rst

```diff
@@ -433,11 +433,8 @@ doubleIndentClassDeclaration
 Default: ``false``
 
 With this set to ``true``, class (and trait / object) declarations
-will be formatted as recommended_ by the Scala Style Guide. That is,
-if the declaration section spans multiple lines, it will be formatted
-so that either the parameter section or the inheritance section is
-doubly indented. This provides a visual distinction from the members
-of the class. For example:
+which span multiple lines, it will be formatted so that either the
```

Comment:
(mostly a note to myself)

I think you want to retain the old doc here, since you haven't changed the behavior of this flag.

You should mention that `multilineConstructorArgumentsDeclaration` overrides this.